packages/devtools_extensions/lib/src/template/extension_manager.dart

```diff
@@ -167,20 +171,40 @@ class ExtensionManager {
           return VmService(
             inStream,
             writeMessage,
+            wsUri: connectedUri.toString(),
           );
         },
       );
       await serviceManager.vmServiceOpened(
         vmService,
         onClosed: finishedCompleter.future,
       );
+      _updateQueryParameter('uri', serviceManager.service!.wsUri!);
     } catch (e) {
-      // TODO(kenz): post a notification to DevTools for errors
-      // or create an error panel for the extensions screens.
-      print('Unable to connect to VM service at $vmServiceUri: $e');
+      final errorMessage =
+          'Unable to connect extension to VM service at $vmServiceUri: $e';
+      showNotification('Error: $errorMessage');
+      _log.shout(errorMessage);
     }
   }
 
+  void _setThemeForValue(String? themeValue) {
+    // Default to use dark theme if [value] is null.
```

Comment:
Do we always default to dark theme? 
Should it be a constant useDarkThemeDefault somewhere in shared constants?

Reply:
Good idea. Done.